src/services/product.service.ts

```diff
@@ -35,8 +36,7 @@ export const getSearchResults = async (
 
 export const postOrder = async (
   products: PostProductOrder[]
-): Promise<Product[]> => {
+): Promise<Order> => {
   const { data } = await axios.post(API_POST_ORDER, { products });
-  console.log(data);
   return data;
 };
```

Comment:
I know this found it's way into this file in an earlier PR but product.services.ts is for products, there should be a separate one for orders